devicetypes/smartthings/zwave-door-window-sensor.src/zwave-door-window-sensor.groovy

```diff
@@ -93,6 +97,13 @@ def updated() {
 	response(cmds)
 }
 
+/**
+ * PING is used by Device-Watch in attempt to reach the Device
+ * */
+def ping() {
```

Comment:
One way to check this is thru Sumo Logic Device-Watch logs. If battery didn't change on graph, it won't result in an event that is visible. 
`_sourceCategory=app-production _source=devicewatch "<deviceId>"`